pkg/apis/minio.min.io/v1/helper.go

```diff
@@ -597,34 +594,11 @@ func (t *Tenant) NewMinIOAdmin(minioSecret map[string][]byte) (*madmin.AdminClie
 	return madmClnt, nil
 }
 
-// CreateConsoleUser function creates an admin user
+// CreateConsoleUser function creates an admin users
 func (t *Tenant) CreateConsoleUser(madmClnt *madmin.AdminClient, userCredentialSecrets []*corev1.Secret, skipCreateUser bool) error {
 	// add user with a 20 seconds timeout
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*20)
 	defer cancel()
-	// Create policy
```

Comment:
we still need this policy, else the call at  setpolicy will fail

Reply:
It won't server already carries this policy @dvaldivia we don't need to set it manually.